install.sh

```diff
@@ -518,11 +527,11 @@ setup_selinux() {
         rpm_target=coreos
         rpm_site_infix=coreos
         package_installer=rpm-ostree
-    elif [ "${VERSION_ID%%.*}" = "7" ]; then
+    elif isNumber "${VERSION_ID}" && [ "${VERSION_ID%%.*}" = "7" ]; then
```

Comment:
Standard format is underscores for sh functions. Also this addresses https://github.com/k3s-io/k3s/issues/8723 case.
```suggestion
    elif is_number "${VERSION_ID:-}" && [ "${VERSION_ID%%.*}" = "7" ]; then
```